sympy/solvers/ode/single.py

```diff
@@ -852,5 +852,80 @@ def _get_general_solution(self, *, simplify: bool = True):
         return [Eq(lhs, C2 + x), Eq(lhs, C2 - x)]
 
 
+class Lienard(SinglePatternODESolver):
+    r"""
+    The general form of the Lienard ODE:
+
+    .. math :: y''(x) + f(x)*y'(x) + y(x)
```

Comment:
This should be an equation.

Reply:
`.. math :: y''(x) + f(x)*y'(x) + y(x) = 0` will this work?